src/index.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import booksReducer from './reducers/book';
 
+const store = createStore(booksReducer);
+console.log(store.getState());
```

Comment:
- [ ] Kindly remove this console.log. Debugging code shouldn't be pushed to production.